.github/workflows/ci.yml

```
@@ -8,16 +8,16 @@ on:
 
 jobs:
   test:
-    name: 'Test'
+    name: Test with node v${{ matrix.node_version }}
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: ['16.13.0']
+        node_version: ['14.17.0', '16']
```

You may want to leave this at 16.13.0 as well

```suggestion
        node_version: ['14.17.0', '16.13.0']
```